components/Credits/Credits.js

```diff
@@ -0,0 +1,23 @@
+import { GitHub, Twitter } from '@mui/icons-material';
+import styles from './Credits.module.css';
+
+export function Credits() {
+  return (
+    <div className={styles.contentContainer} id="credits">
+      <h1 className={styles.title}>Mente</h1>
+      <div className={styles.iconsContainer}>
+        <a href="https://github.com/alshakerM" target="_blank" rel="noreferrer">
```

Comment:
This should lead to the repo of the project, not your profile.

Reply:
On it